docs/content/concepts/flow-control/label/classifier.md

```diff
@@ -3,5 +3,108 @@ title: Flow Classifier
 ---
 
 :::info
-See also [Classifier reference](/reference/configuration/policies.md#-v1classifier)
+See also [Classifier reference][reference]
 :::
+
+If existing [flow labels](label.md) are not sufficent,
+Flow Classifier can be used to inject additional ones
+without any changes to your service.
+
+A classifier is defined as a resource in a [policy][policies]
+and describes a set of _rules_ on how to create new flow labels based
+on request metadata. Aperture uses Envoy's [External Authorization][ext-authz]
+definition to describe the request metadata (more specifically, the
+[AttributeContext][attr-context]). The [INPUT section at this Rego
+playground][rego-playground] is an example how the request attributes may look like.
+
+:::note
+Classifiers are available only on _traffic_ control points. That's because they
```

Comment:
Labels can be provided directly by developers for feature control points.

Reply:
Yeah, that's what I've written in next sentence. Do you think it needs some rewording?

Reply:
It sounded like a limitation of feature control points — we can reword that features already have labels provided directly by developers for each request

Reply:
Agreed. Will do